app/controllers/users/rdvs_controller.rb

```diff
@@ -44,6 +44,7 @@ def create
       set_user_name_initials_verified
       redirect_to users_rdv_path(@rdv, invitation_token: notifier.rdv_users_tokens_by_user_id[current_user.id]), notice: t(".rdv_confirmed")
     else
+      # TODO: cette liste de paramètres devrait ressembler a SearchController#search_params, mais sans certains paramètres de choix du wizard de crenau
```

Comment:
j'ai fais une typo 😬 
```suggestion
      # TODO: cette liste de paramètres devrait ressembler a SearchController#search_params, mais sans certains paramètres de choix du wizard de créneaux
```